miner/miner.go

```diff
@@ -334,14 +346,45 @@ func (m *Miner) computeTicket(ctx context.Context, addr address.Address, base *M
 	}, nil
 }
 
-func (m *Miner) createBlock(base *MiningBase, addr address.Address, ticket *types.Ticket, proof *types.EPostProof) (*types.BlockMsg, error) {
+func (m *Miner) actorLookup(ctx context.Context, addr address.Address, ts *types.TipSet) (uint64, *types.BigInt, error) {
+	// TODO: strong opportunities for some caching in this method
+	act, err := m.api.StateGetActor(ctx, addr, ts)
+	if err != nil {
+		return 0, nil, xerrors.Errorf("looking up actor failed: %w", err)
+	}
 
-	pending, err := m.api.MpoolPending(context.TODO(), base.ts)
+	msgs, err := m.api.ChainGetTipSetMessages(ctx, ts.Key())
```

Comment:
so i think we can remove this whole messages checking. its just a sanity check at this point, and i think we found out the real issue separately